apps/test-viewer/src/UiProvidersConfig.tsx

```diff
@@ -48,8 +49,19 @@ const configuredUiItems = new Map<string, UiItem>([
   [
     "tree-widget",
     {
-      initialize: async () => TreeWidget.initialize(),
-      createUiItemsProvider: () => new TreeWidgetUiItemsProvider(),
+      initialize: async () => {
+        TreeWidget.initialize();
```

Comment:
dont these need to be awaited?

Reply:
they are once they are once passed into the getUiProvidersConfig function

Reply:
refer to https://github.com/iTwin/viewer-components-react/pull/427#discussion_r1170802779